test/client/fixtures/core/dom-utils-test.js

```diff
@@ -0,0 +1,43 @@
+var hammerhead   = window.getTestCafeModule('hammerhead');
+var browserUtils = hammerhead.utils.browser;
+
+asyncTest('isIFrameWindowInDOM', function () {
+    expect(browserUtils.isIE ? 2 : 1);
+
+    var messageCounter = 0;
+
+    function finishTest () {
+        window.removeEventListener('message', onMessage, false);
+        start();
+    }
+
+    function onMessage (event) {
+        if (messageCounter === 0) {
+            equal(event.data, true);
+
+            var iFramePostMessage = $iFrame[0].contentWindow.postMessage.bind($iFrame[0].contentWindow);
```

Comment:
Add empty line after `var` declaration. Linting is not enabled here now but we will do it